packages/mobile/src/firebase/saga.ts

```diff
@@ -81,18 +83,18 @@ function* initializeFirebase() {
   }
 }
 
-export function* syncLanguageSelection({ language }: SetLanguage) {
+export function* syncLanguageSelection(action: PayloadAction<string>) {
```

Comment:
I propose we do the following to avoid the type to diverge from the original action:
```suggestion
export function* syncLanguageSelection(action: ReturnType<typeof setLanguage>) {
```